Formula/m/mlc.rb

```diff
@@ -0,0 +1,37 @@
+class Mlc < Formula
+  desc "Check for broken links in markup files"
+  homepage "https://github.com/becheran/mlc"
+  url "https://github.com/becheran/mlc/archive/refs/tags/v0.22.0.tar.gz"
+  sha256 "18d22c96cf2fccd6937268db141c74f24fa3113d21ac55452d8a7eb05150b489"
+  license "MIT"
+  head "https://github.com/becheran/mlc.git", branch: "master"
+
+  depends_on "rust" => :build
+  depends_on "openssl@3" # `openssl-sys` crate
```

Comment:
better with this? no linkage for openssl on macos builds
```suggestion
  depends_on "pkgconf" => :build
  depends_on "rust" => :build

  on_linux do
    depends_on "openssl@3"
  end
```